dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -37,6 +38,11 @@
 
 from dftimewolf.lib.recipes import manager as recipes_manager
 from dftimewolf.lib.state import DFTimewolfState
+from dftimewolf.lib import logging_utils
+
+logger = logging.getLogger('dftimewolf')
+DEFAULT_LOG_FILE = os.path.join('/', 'tmp', 'dftimewolf.log')
```

Comment:
what about using `os.sep` instead of `/` here?

Reply:
Sure, even though I'm pretty sure this will fail on an OS where the `sep` is not `/`.